omero/sysadmins/admins-with-restricted-privileges.rst

```diff
@@ -17,9 +17,13 @@ facility managers, image analysts, or anybody who needs to organize
 users and data of others in OMERO.
 
 Full administrators in OMERO can create new administrators with
-restricted privileges using the OMERO.web
-interface, see the :help:`create new users <sharing-data#admin>` section
-of our Help documentation.
+restricted privileges using the OMERO.web interface, see the
+:help:`create new users <sharing-data#admin>` section of our Help
+documentation. OMERO.cli does not yet support easy management of
+restrictions nor does it offer the helpful :doc:`permissions mapping
+<mapping-restricted-admins>` so we do not recommend :doc:`using
```

Comment:
Because there could be a mis-configuration or because it's more complicated? If the latter, perhaps this could be phrased as "advanced use".